packages/introspect/src/tests/pg_types.test.ts

```
@@ -0,0 +1,285 @@
+                    'postgresql://postgres:postgres@localhost:5432/postgres',
+                includeSchemas: ['public'],
+                defaultSchema: 'public',
+                includeTables: [tableName],
+                outDir: '../introspect/src/tests/generated',
+            });
+
+            for (const [name, value] of Object.entries(await selectData())) {
+                const columnType = (schema.properties?.[tableName] as any)
+                    ?.properties?.columns?.properties?.[name]?.properties?.type;
+
+                delete columnType?.id;
```

Biome

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)